projects/packages/forms/src/blocks/contact-form/editor.scss

```diff
@@ -612,10 +612,11 @@
 .jetpack-field-checkbox,
 .jetpack-field-consent {
 	display: flex;
+}
 
-	.jetpack-field-label .jetpack-field-label__input {
-		font-weight: 400;
-	}
+.jetpack-field-consent,
+.jetpack-field-checkbox .jetpack-field-label .jetpack-field-label__input {
```

Comment:
From what I can tell this selector isn't doing anything. I guess it's the same selector that was used in trunk, but what I'm not sure about is if it's working

It's very high specificity, so if it did work it'd probably break global styles. 😅 

I think we can either remove it, or fix it and change it to 0-1-0 specificity

Reply:
Just for clarification, it's the `.jetpack-field-checkbox .jetpack-field-label .jetpack-field-label__input` one that I'm referring to.